core/src/components/progress-bar/progress-bar.ionic.scss

```
@@ -0,0 +1,31 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./progress-bar";
```

The ionic styling files need to use `@use` since `@import` has been deprecated. New style files are to use `@use` to make it easier on the team to migrate in the future.

```suggestion
@use "./progress-bar";
```